### ISSUE TITLE
add event messages and prerequisites

### ISSUE THREAD

**Issue by ozm8** [FlyingPikachu/Quests#10]
Hello Can you make it so you can add messages after a player does a quest so for example a player has a place objective then you want to put a message saying now you need to return to bon npc for your reward.
Also prerequisites so we can define which quest needs to be done before each quest

**Comment by QueenKisaki** [comment]
There are 'start' and 'complete' messages you can use for the quest itself and for each stage of the quest. You can also use the quest requirements to set prerequisites.

**Comment by ozm8** [comment]
ok. also can you make the objective update. atm when you have say an objective to kill 2 zombies it will say 0/2 kill zombies or what ever. make it so when you kill just 1 of those zombies it'll come up and say 1/2 zombies

**Comment by ozm8** [comment]
oh also just found out when u make an place oak wooden door objective it doesn't work. it doesn't go to the next objective i had this same problem with the last questing plugin i used too with placed doors

**Comment by FlyingPikachu** [comment]
@ozm8 Your original question has been answered by @QueenKisaki 

Please open a new issue if your wooden door problem persists. Thanks!
